mcmc/MCMCProcessor.h

```diff
@@ -77,21 +80,21 @@ class MCMCProcessor {
     void ResetHistograms();
     
     // Get the number of parameters
-    int GetNParams() { return nDraw; };
-    int GetNFlux() { return nFlux; };
-    int GetNXSec() { return nParam[kXSecPar]; };
-    int GetNND() { return nParam[kND280Par]; };
-    int GetNFD() { return nParam[kFDDetPar]; };
-    int GetOSC() { return nParam[kOSCPar]; };
+    inline int GetNParams() { return nDraw; };
```

Comment:
I'm guessing these are all just inline for a small speed increase?

Reply:
Exactly